src/Voxel.MiddyNet/MiddyNetContext.cs

```diff
@@ -35,5 +37,11 @@ public void AttachToLambdaContext(ILambdaContext context)
         }
 
         public Func<ILambdaLogger, IMiddyLogger> LoggerFactory { get; }
+
+        internal void FinishedBeforeMiddlewares()
+        {
+            MiddlewareAfterExceptions = new List<Exception>();
+            MiddlewareBeforeExceptions = null; // We assume that the function has handled the exceptions Before middlewares might have thrown
```

Comment:
I believe we talked about keeping the exceptions in case the after middlewares need it?

Reply:
mm that would change things. ¿can we confirm?